core/controllers/admin.py

```diff
@@ -203,6 +208,26 @@ def _reload_collection(self, collection_id):
                 feconf.SYSTEM_COMMITTER_ID, unicode(collection_id))
         else:
             raise Exception('Cannot reload a collection in production.')
+    def _reload_dummy_exploration(self, dummy_exp_count):
```

Comment:
Add newline above this one. In general, try to follow the existing style as closely as possible.